recipes/spyre/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set version = "0.2.1" %}
+{% set commit = "ba37d1b6327afbc2c7cf8e7b8ef1d9a6fbd17d76" %}
+
+package:
+    name: spyre
+    version: {{ version }}.dev0
```

Comment:
Ouch. Is it planned for there to be releases? I want to avoid pinning of commit SHAs as much as possible.